Core/HLE/sceKernelThread.cpp

```diff
@@ -2148,11 +2148,16 @@ void __KernelReturnFromThread()
 	// The stack will be deallocated when the thread is deleted.
 }
 
-void sceKernelExitThread(int exitStatus) {
+int sceKernelExitThread(int exitStatus) {
+	if (!__KernelIsDispatchEnabled() && sceKernelGetCompiledSdkVersion() > 0x0307FFFF)
+		return ERROR_KERNEL_WAIT_CAN_NOT_WAIT;
```

Comment:
Let's use `return hleLogError(SCEKERNEL, ERROR_KERNEL_WAIT_CAN_NOT_WAIT);`.  This way it will show in the log and we won't get confused if this breaks another game.

-[Unknown]